sdk/machinelearningservices/Azure.ResourceManager.MachineLearningServices/src/autorest.md

```diff
@@ -0,0 +1,13 @@
+# Generated code configuration
+
+Run `dotnet build /t:GenerateCode` to generate code.
+
+``` yaml
+
+azure-arm: true
+require: https://raw.githubusercontent.com/forteddyt/azure-rest-api-specs/db218390cc57f2509db0e88d8d5ffd1872aa9435/specification/machinelearningservices/resource-manager/readme.md
```

Comment:
Is this intentionally not in master? I believe you will encounter issues with auto-generation if it is in a feature branch.

Reply:
This is intentional change from a feature branch. The swaggers from spec repo are not merged to master yet.